src/Config/Level/CodeQualityLevel.php

```diff
@@ -164,6 +165,7 @@ final class CodeQualityLevel
         CleanupUnneededNullsafeOperatorRector::class,
         DisallowedEmptyRuleFixerRector::class,
         ConvertStaticPrivateConstantToSelfRector::class,
+        ReplaceProductionConstantsWithLiteralsInTestFilesRector::class,
```

Comment:
The rule you created is configurable for user that need change specific constant to literal, no need to register it on code quality level, revert this ;)